makefiles/info.inc.mk

```
@@ -3,7 +3,8 @@
         info-features-provided info-features-required \
         info-features-used \
         info-debug-variable-% info-toolchains-supported \
-        check-toolchain-supported
+        check-toolchain-supported \
+        info-programmers-supported
```

```suggestion
        info-programmers-supported \
        #
```

will make future change easier.

---

fixed